render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlCoalesceSerializerTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl.JpqlCoalesce
+import com.linecorp.kotlinjdsl.render.TestRenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializerTest
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import io.mockk.impl.annotations.MockK
+import io.mockk.verifySequence
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+@JpqlSerializerTest
+class JpqlCoalesceSerializerTest : WithAssertions {
+    private val sut = JpqlCoalesceSerializer()
+
+    @MockK
+    private lateinit var writer: JpqlWriter
+
+    @MockK
+    private lateinit var serializer: JpqlRenderSerializer
+
+    private val expression1 = Expressions.nullValue<Int>()
+    private val expression2 = Expressions.value(2)
+    private val expression3 = Expressions.value(3)
+
+    @Test
+    fun handledType() {
+        // when
+        val actual = sut.handledType()
+
+        // then
+        assertThat(actual).isEqualTo(JpqlCoalesce::class)
+    }
+
+    @Test
+    fun `serialize if others input does not entered then draws only expression1, 2`() {
```

Comment:
The name of the test is different from the other tests. Please rename it based on the other tests. 

And it looks like the tests for this class should only need to have one `serialize()`. It doens't need to test two `emptyList()` and two `listOf()`.

Reply:
thanks. review applied.thanks. review applied.